### PR TITLE
LG-3031 Change AUTH_URL config to AAMVA_AUTH_URL

### PR DESCRIPTION
**Pull request by jmhooper**
**Why**: The of the config values in this repo match the names that we use in the IdP. In the IdP config, it is not obvious that `AUTH_URL` is the config for the AAMVA auth URL.

I changed the URLs we use in the unit tests to not match the defaults. I did this to make sure that I did not drop a config somewhere and it wasn't falling back to the defaults.

### CODE REVIEW

**Comment by jmhooper** [comment]
I have a rubucop linter in my editor that autofixes files when I save. That caused a little bit of noise in the diff here.

File: lib/aamva/request/authentication_token_request.rb [hunk 1]
```diff
@@ -34,15 +34,15 @@ def initialize(
...
-      rescue Faraday::TimeoutError, Faraday::ConnectionFailed => err
-        message = "AAMVA raised #{err.class} waiting for authentication token response: #{err.message}"
+      rescue Faraday::TimeoutError, Faraday::ConnectionFailed => e
+        message = "AAMVA raised #{e.class} waiting for authentication token response: #{e.message}"
         raise ::Proofer::TimeoutError, message
       end
 
       def self.auth_url
-        Env.fetch('AUTH_URL', DEFAULT_AUTH_URL)
+        Env.fetch('AAMVA_AUTH_URL', DEFAULT_AUTH_URL)
```
Comment (zachmargolis) [inline lib/aamva/request/authentication_token_request.rb:45]: should we have it also fall back on the old name and warn?

File: lib/aamva/request/security_token_request.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ def send
       end
 
       def self.auth_url
-        Env.fetch('AUTH_URL', DEFAULT_AUTH_URL)
+        Env.fetch('AAMVA_AUTH_URL', DEFAULT_AUTH_URL)
```
Comment (zachmargolis) [inline lib/aamva/request/security_token_request.rb:39]: ditto about falling back on the old URL

File: lib/aamva/request/authentication_token_request.rb (hunk 1 above)
Comment (jmhooper) [reply lib/aamva/request/authentication_token_request.rb:45]: We can but we actually don't use the old name anywhere. And if we did change it it wouldn't work. The default is hardcoded somewhere else in the code. PR to fix that is coming up next.

**Comment by jmhooper** [comment]
I'm planning to bump the version once I replace the hardcoded URLs.

File: lib/aamva/request/authentication_token_request.rb (hunk 1 above)
Comment (zachmargolis) [reply lib/aamva/request/authentication_token_request.rb:45]: Ok sounds good!

**Pull request merged**